shell/platform/darwin/macos/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -431,30 +419,22 @@ - (bool)testAutocompleteEnabledWhenAutofillSet {
       [[FlutterTextInputPlugin alloc] initWithViewController:viewController];
 
   // Set input client 1.
-  [plugin handleMethodCall:[FlutterMethodCall
-                               methodCallWithMethodName:@"TextInput.setClient"
-                                              arguments:@[
-                                                @(1), @{
-                                                  @"inputAction" : @"action",
-                                                  @"inputType" : @{@"name" : @"inputName"},
-                                                  @"autofill" : @{
-                                                    @"uniqueIdentifier" : @"field1",
-                                                    @"hints" : @[ @"name" ],
-                                                    @"editingValue" : @{@"text" : @""},
-                                                  }
-                                                }
-                                              ]]
+  NSDictionary* setClientConfig = @{
+    @"inputAction" : @"action",
+    @"inputType" : @{@"name" : @"inputName"},
+    @"autofill" : @{
+      @"uniqueIdentifier" : @"field1",
+      @"hints" : @[ @"name" ],
+      @"editingValue" : @{@"text" : @""},
+    }
+  };
+  [plugin handleMethodCall:[FlutterMethodCall methodCallWithMethodName:@"TextInput.setClient"
+                                                             arguments:@[ @(1), setClientConfig ]]
                     result:^(id){
                     }];
 
   // Verify autocomplete is enabled.
   EXPECT_TRUE([plugin isAutomaticTextCompletionEnabled]);
-
-  // Verify content type is nil for unsupported content types.
-  if (@available(macOS 11.0, *)) {
-    EXPECT_EQ([plugin contentType], nil);
-  }
```

Comment:
Just to confirm, is this deletion intentional?

Reply:
Good catch - no! That code showed up about 50 lines down below and caused another test to fail; on looking at it there, I was thinking "why would I add this?!" Answer: apparently an inadvertent cut/paste!